modules/configuring-the-model-registry-component.adoc

```diff
@@ -0,0 +1,124 @@
+ifdef::cloud-service[]
+* You have installed the Red Hat Authorino, Red Hat OpenShift Serverless, Red Hat OpenShift Service Mesh, and Red Hat OpenShift AI Operators.
+* You have sufficient resources. For more information about the minimum resources required to use {productname-short}, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-and-deploying-openshift-ai_install[Installing and deploying {productname-short}].
+endif::[]
+ifdef::self-managed[]
+* You have installed the Red Hat Authorino, Red Hat OpenShift Serverless, Red Hat OpenShift Service Mesh, and Red Hat OpenShift AI Operators.
+* You have sufficient resources. For more information about the minimum resources required to use {productname-short}, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-and-deploying-openshift-ai_install[Installing and deploying {productname-short}] (for disconnected environments, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/deploying-openshift-ai-in-a-disconnected-environment_install[Deploying {productname-short} in a disconnected environment]).
+endif::[]
+ifdef::upstream[]
+* You have installed the Red Hat Authorino, Red Hat OpenShift Serverless, Red Hat OpenShift Service Mesh, and Open Data Hub Operators.
+* You have sufficient resources. For more information about the minimum resources required to use {productname-short}, see link:{odhdocshome}/installing-open-data-hub/#installing-the-odh-operator-v2_installv2[Installing the {productname-short} Operator version 2].
+endif::[]
+
+.Procedure
+. In the {openshift-platform} console, click *Operators* -> *Installed Operators*.
+ifdef::self-managed,cloud-service[]
+. Search for the *Red Hat OpenShift AI Operator* version 2.14+, and then click the Operator name to open the Operator details page.
+endif::[]
+ifdef::upstream[]
+. Search for the *Open Data Hub Operator*, and then click the Operator name to open the Operator details page.
+endif::[]
+. Click the *Data Science Cluster* tab.
+. Click the default instance name (for example, *default-dsc*) to open the instance details page.
+. Click the *YAML* tab to show the instance specifications.
+ifdef::upstream[]
+. Find the `spec:components` section, and then add or update it to include the following `modelregistry` component entry, with the `managementState` field set to `Managed`, and the `registriesNamespace` field set to `odh-model-registries`:
++
+[source]
+----
+ modelregistry:
+    managementState: Managed
+    registriesNamespace: odh-model-registries
+----
+endif::[]
+ifndef::upstream[]
+. Find the `spec:components` section, and then add or update it to include the following `modelregistry` component entry, with the `managementState` field set to `Managed`, and the `registriesNamespace` field set to `rhoai-model-registries`:
```

Comment:
Same comment here, `spec.components` instead of `spec:components`

Reply:
Updated, thank you!